lnrpc/routerrpc/router.proto

```diff
@@ -981,6 +1008,9 @@ message ForwardHtlcInterceptRequest {
     // The block height at which this htlc will be auto-failed to prevent the
     // channel from force-closing.
     int32 auto_fail_height = 10;
+
+    // The custom records of the peer's incoming p2p wire message.
+    map<uint64, bytes> in_wire_custom_records = 11;
```

Comment:
No, unfortunately not. Because the `HtlcInterceptor` only fires for _forwarded_ packages. But Carol is the destination hop here. But we do test something similar in `testForwardInterceptorWireRecords` where we have a 4 node network.